app/code/Dev/ProductComment/Model/Attribute/Backend/Comment.php

```diff
@@ -0,0 +1,27 @@
+<?php
+/**
+ * Copyright © 2016 Magento. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Dev\ProductComment\Model\Attribute\Backend;
+
+class Comment extends \Magento\Eav\Model\Entity\Attribute\Backend\AbstractBackend
+{
+    /**
+     * Validate
+     * @param \Magento\Catalog\Model\Product $object
+     * @throws \Magento\Framework\Exception\LocalizedException
+     * @return bool
+     */
+    public function validate($object)
+    {
+        $value = $object->getData($this->getAttribute()->getAttributeCode());
```

Comment:
This is a class is called a backend model which validates attribute value on save.

In your case, this backend model is not needed because it actually does notihgn at all and can be safely removed from code and attribute.